web/src/lib/components/photos-page/asset-grid.svelte

```diff
@@ -427,7 +427,9 @@
 <!-- Right margin MUST be equal to the width of immich-scrubbable-scrollbar -->
 <section
   id="asset-grid"
-  class="scrollbar-hidden h-full overflow-y-auto outline-none pb-[60px] {isEmpty ? 'm-0' : 'ml-4 tall:ml-0 mr-[60px]'}"
+  class="scrollbar-hidden h-full overflow-y-auto outline-none pb-[60px] {isEmpty
+    ? 'm-0'
+    : 'ml-4 tall:ml-0 mr-[20px] md:mr-[60px]'}"
```

Comment:
Just so you know, this component is also used on other pages, such as the main timeline. Reducing the margin causes the asset grid and scrollbar to overlap and when thumbnails have a light color, the scrollbar labels have poor contrast. Maybe we can add a background color to the labels?

| Current | With background |
| - | - |
| ![scrollbar-without-bg](https://github.com/user-attachments/assets/859269e9-5605-439a-ad69-74aacbe0e62a) | ![scrollbar-with-bg](https://github.com/user-attachments/assets/9f9563c8-0242-4a3d-b8a3-79dac806bc39) |

Reply:
Looks good to me. In *mnbm* Photos they solved it by having the scollbar labels disappear after a short timeout (might try to replicate that in a seperate PR). I've enabled edits by maintainers, I think you can push your changes to my branch.